File: Java/Typing/Precedence.java
class Main {
	public static void main (String[] args) {
		System.out.println(123 * 100.0 / 123);
		System.out.println(123 * (100.0 / 123));
		System.out.println((double)123 * 100.0 / 123);		
		System.out.println(123 * 100.0 / (double)123);
		System.out.println(123 * 100 / 123);
		System.out.println(123 / 123 * 100.0);
		System.out.println(123 / (123 * 100.0));
		System.out.println((double)123 / 123 * 100.0);
		System.out.println(123 / (double)123 * 100.0);
		System.out.println(123 / 123 * 100);
	}
}

//https://pt.stackoverflow.com/q/442111/101

File: CSharp/Operator/Precedence.cs
using System;

public class C {
    public static int Main() {
        var conta = new Conta();
        var txtValor = new Form();
        bool retorno;
 		if ((retorno = conta.Saca(Convert.ToDouble(txtValor.Text)))) {
            Console.WriteLine(retorno);
            return 1;
        }
        return 0;
     }
}

public class Conta {
    public bool Saca(double x) => true;
}

public class Form {
    public String Text;
}

//https://pt.stackoverflow.com/q/206902/101
